Lesson05/HW05.IntuitionTraining/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+
+namespace HW05.IntuitionTraining
+{
+    internal class Program
+    {       
+        public static object Guess { get; private set; }
+
+        static void Main(string[] args)
+        {
+            while(true)
+            {
+                Random random = new Random();
+                int returnValue = random.Next(1, 5);
+                int Guess = 0;
```

Comment:
The second time I say. Read about variable naming at Microsoft documentation...

Reply:
done